app/players/player.py

```diff
@@ -0,0 +1,15 @@
+from abc import ABC, abstractmethod
+
+
+class Player(ABC):
+
+    def __init__(self, nickname: str) -> None:
+        self.nickname = nickname
+
+    @abstractmethod
+    def get_rating(self) -> None:
+        pass
+
+    @abstractmethod
+    def player_info(self) -> None:

```

Comment:
you should add correct annotaions to abstract methods. If you know what type you want to return in classes that will inherit from this one, you should provide accurate annotations already here